packages/components/src/modal/test/index.tsx

```diff
@@ -261,115 +290,70 @@ describe( 'Modal', () => {
 				originalGetClientRects;
 		} );
 
-		it( 'should focus the first focusable element in the contents (if found) when `firstContentElement` passed as value for `focusOnMount` prop', async () => {
+		it( 'should focus the Modal dialog by default when `focusOnMount` prop is not provided', async () => {
 			const user = userEvent.setup();
 
-			const FocusMountDemo = () => {
-				const [ isShown, setIsShown ] = useState( false );
-				return (
-					<>
-						<button onClick={ () => setIsShown( true ) }>📣</button>
-						{ isShown && (
-							<Modal
-								focusOnMount="firstContentElement"
-								onRequestClose={ () => setIsShown( false ) }
-							>
-								<p>Modal content</p>
-								<a
-									href="https://wordpress.org"
-									data-testid="first-focusable-element"
-								>
-									First Focusable Element
-								</a>
+			render( <FocusMountDemo focusOnMount={ true } /> );
```

Comment:
Removing `focusOnMount` prop as per test description

```suggestion
			render( <FocusMountDemo /> );
```

Reply:
How did I miss this? 🤦